src/app/components/list/view-switcher/index.html

```diff
@@ -21,44 +21,44 @@
       <h3 class="sky-text-success">Use when</h3>
         <sky-docs-visual-example>
           <sky-docs-visual-example-text>
-              <p>Use the list view switcher when the user needs to switch between different views of the same data.</p>
+              <p>Use list view switchers when users need to switch between different views of the same data.</p>
           </sky-docs-visual-example-text>
           <sky-docs-visual-example-image>
             <div>
               <img src="~/assets/img/guidelines/list-view-switcher/use-when.png" />
```

Comment:
@Blackbaud-ToddRoberts, the image here illustrates when two switcher buttons are in place, but it doesn’t illustrate the different views. Would it be useful to include a second image with the other button selected to show the difference in the views?

Reply:
Discussed this with @Blackbaud-AdamFunderburk, our thought is that the demo should be sufficient for illustrating this.

Reply:
Sounds good. Thanks.